backend/cmd/web/rentACarHandler.go

```diff
@@ -24,6 +24,25 @@ func (app *Application) GetRentACarCompanies(w http.ResponseWriter, r *http.Requ
 	}
 }
 
+func (app *Application) CreateRentACarCompany(w http.ResponseWriter, r *http.Request) {
+	var rac models.RentACarCompany
+
+	err := json.NewDecoder(r.Body).Decode(&rac.RentACarCompanyProfile)
+	if err != nil {
+		app.ErrorLog.Println("Could not decode JSON")
+		w.WriteHeader(http.StatusInternalServerError)
+		return
+	}
+
+	err = app.Store.CreateRentACarCompany(&rac)
+	if err != nil {
+		app.ErrorLog.Printf("Could not add hotel to database")
```

Comment:
Pogresna poruka za gresku.
Valjda treba "Could not add rent a car company to database" ?

Reply:
Slazem se kolega